heroku_connect/db/models/base.py

```diff
@@ -1,12 +1,26 @@
 from django.core import checks
-from django.db import models
+from django.db import NotSupportedError, models
 
 from . import fields
 from ...conf import settings
 
 __all__ = ('HerokuConnectModel',)
 
 
+class HerokuConnectModelManager(models.Manager):
+    def update(self, **kwargs):
+        if self.model.sf_access == 'read_only':
+            raise NotSupportedError(
+                'Save/Update operation is not allowed for ReadOnly model')
```

Comment:
ether `for a read-only model` or `for read-only models`
Please also add a trailing full stop.